code/modules/mailorder/modr_entries.dm

```diff
@@ -0,0 +1,365 @@
+		desc = "Rich tropical flavor in a spiky package, express-delivered."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/pineapple)
+		cost = PAY_TRADESMAN / 3
+
+	pumpkin
+		name = "Pumpkin"
+		desc = "Shipped whole and intact - perfect for autumnal classics."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/pumpkin)
+		cost = PAY_TRADESMAN / 4
+
+	catnip
+		name = "Catnip Sprig"
+		desc = "Pure nepeta cataria, perfect for your feline pals."
+		order_items = list(/obj/item/plant/herb/catnip)
+		cost = PAY_TRADESMAN / 5
+
+	mint
+		name = "Mint Sprig"
+		desc = "Effervescent mint, carefully preserved for peak flavor."
+		order_items = list(/obj/item/plant/herb/mint)
+		cost = PAY_TRADESMAN / 5
+
+	cereal_pope
+		name = "Pope Crunch Cereal"
+		desc = "Made with only the highest-quality iron supplements."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_wonks
+		name = "Honey Wonks Cereal"
+		desc = "Your sticky-sweet sugar rush, at a super special price."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/honey)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_monki
+		name = "Tanh-O-Nys Cereal"
+		desc = "Does it taste like banana? Who knows! Buy some and find out."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/tanhony)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_roach
+		name = "Roach Puffs Cereal"
+		desc = "Everybody's favorite chocolate-themed roach-themed treat."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/roach)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_bundle
+		name = "Munch 'Em All Bundle"
+		desc = "Can't decide on a cereal? Buy them all and save!"
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box,
+		/obj/item/reagent_containers/food/snacks/cereal_box/honey,
+		/obj/item/reagent_containers/food/snacks/cereal_box/tanhony,
+		/obj/item/reagent_containers/food/snacks/cereal_box/roach)
+		cost = PAY_UNTRAINED
+
+
+//Survival Mart: primarily medical, but not exclusively
+ABSTRACT_TYPE(/datum/mail_order/survmart)
+/datum/mail_order/survmart
+
+	oxytank
+		name = "Emergency Oxygen Tank"
+		desc = "An easy-to-use compact gas tank pre-loaded with pure oxygen."
+		order_items = list(/obj/item/tank/emergency_oxygen)
+		cost = PAY_TRADESMAN
+
+	wintercoat
+		name = "Winterized Overcoat"
+		desc = "Beat the cold of space with your very own cozy coat."
+		order_items = list(/obj/item/clothing/suit/wintercoat)
+		cost = PAY_TRADESMAN / 2
+
+	cleangear
+		name = "Medical Wear Bundle"
+		desc = "Contains latex gloves and sterile mask."
+		order_items = list(/obj/item/clothing/gloves/latex,/obj/item/clothing/mask/surgical)
+		cost = PAY_UNTRAINED / 2
+
+	medbelt
+		name = "Medical Belt"
+		desc = "Top-quality belt for rapid medical equipment access."
+		order_items = list(/obj/item/storage/belt/medical)
+		cost = PAY_DOCTORATE
+		order_perm = list(access_medical)
```

Comment:
Disagree with this being an option.

Reply:
I can see how it'd be a bit strong, and as there are alternatives like "jam a box in your backpack" I have no objection to removing it.